src/utils/telemetry.ts

```diff
@@ -85,9 +87,14 @@ export class Telemetry {
     }
   }
 
-  static init(environment: string, allowDataDogLogging = false) {
+  static init(
+    environment: string,
+    ehr: string | undefined = undefined,
```

Comment:
In the ehr-data-integration service we are using "unknown" in place of undefined. It could be good to do the same thing here.
```suggestion
    ehr = "unknown",
```